code/datums/brain_damage/magic.dm

```diff
@@ -95,7 +102,7 @@
 		playsound(owner, 'sound/magic/demon_attack1.ogg', 50)
 		owner.visible_message("<span class='warning'>[owner] is torn apart by invisible claws!</span>", "<span class='userdanger'>Ghostly claws tear your body apart!</span>")
 		owner.take_bodypart_damage(rand(20, 45), wound_bonus=CANT_WOUND)
-	else if(prob(50))
+	else if(DT_PROB(30, delta_time))
```

Comment:
Shouldn't this be 25?

Reply:
The more accurate probabilities aren't exactly half of the original probabilities. I mostly just used desmos to figure them out.

Reply:
To clarify, you solve for x to find the new rate:

1-(1-x)^OLDDT=OLDCHANCE

So in this case:

1-(1-x)^2=0.5
=> x=~0.3

Reply:
good lord